scripts/test/integration-tests.py

```diff
@@ -88,8 +94,8 @@ def test_created_code_compiles(self):
         self.runCreate(project_prefix + '-frontend-mongo', 'BACKEND', with_mongo=True)
         self.runCreate(project_prefix + '-library', 'LIBRARY')
 
-        self.runSbtCommand(projects, 'compile')
```

Comment:
Given that `compile` is implied by `test`, these tests run a bit quicker by not having to spin up the SBT server twice